src/Nest/Indices/MappingManagement/GetMapping/GetMappingResponse.cs

```diff
@@ -44,6 +44,13 @@ public class GetMappingResponse : DictionaryResponseBase<IndexName, IndexMapping
 		[JsonIgnore]
 		public IReadOnlyDictionary<IndexName, IndexMappings> Mappings => Indices;
 
+		[Obsolete("Use GetMappingFor explicitly instead this is a leaky abstraction that returns the mapping of the first index's first type on the response")]
+		public TypeMapping Mapping => this.Indices.FirstOrDefault().Value?.Mappings?.FirstOrDefault().Value;
```

Comment:
Needs a null check. I'm thinking this probably should be the old behavior and not SingleOrDefault as below.